sdk/spring/scripts/compatibility_update_supported_version_matrix_json.py

```diff
@@ -37,7 +45,8 @@ def get_supported_spring_boot_version(filepath):
     for entry in data:
         for key in entry:
             if entry[key] == "SUPPORTED":
-                supported_version_list.append(entry["spring-boot-version"])
+                if entry["spring-boot-version"].startswith(get_args().spring_boot_major_version):
```

Comment:
Would it be better if we change to `startswith(get_args().spring_boot_major_version + '.')`?